packages/server-core/src/user/user-inventory/user-inventory.hooks.ts

```diff
@@ -10,7 +14,37 @@ export default {
     get: [],
     create: [],
     update: [disallow()],
-    patch: [],
+    patch: [
+      async (context: HookContext): Promise<HookContext> => {
+        if (context.data.type === 'Wallet transfer') {
```

Comment:
lets rename this from `Wallet transfer` to `transfer`

Reply:
Changed in the current version